### PR TITLE
[ADD] events: add documentation

### PR DESCRIPTION
**Pull request by kbr-odoo**

### CODE REVIEW

File: events/overview.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Overview
+========
+
+.. toctree::
+   :titlesonly:
+
+   overview/create_event
+   overview/track_attendees
+   overview/tickets
+   overview/track_talks
```
Comment (Lau-Odoo) [inline events/overview.rst:11]:
```suggestion
   overview/create_event
   overview/tickets
   overview/track_attendees
   overview/track_talks
```

Comment (Lau-Odoo) [reply events/overview.rst:11]: You sell tickets before tracking your attendees :smile:

File: events/overview/track_talks.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
+======================
+Track and Manage Talks
+======================
+
+Allow partners to submit talk proposals so they can share their knowledge and expertise, and to add
+extra value to your event.
```
Comment (Lau-Odoo) [inline events/overview/track_talks.rst:6]:
```suggestion
Allow partners to submit talk proposals so they can share their knowledge and expertise. Good speakers and talks also add extra value to your event.
```

File: events/overview/track_talks.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Configuration
+=============
+
+| Go to :menuselection:`Events --> Configuration --> Settings`, and enable *Schedule & Tracks*.
```
Comment (Lau-Odoo) [inline events/overview/track_talks.rst:11]:
```suggestion
| Go to :menuselection:`Events --> Configuration --> Settings` and enable *Schedule & Tracks*.
```

File: events/overview/track_talks.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+| Now, in your event's form, once *Website menu* is enabled, the options *Tracks on Website* and
+  *Track Proposals on Website* become available. Enabling them add the menus *Talks*,
+  *Talk Proposals*, and *Agenda* to your website.
```
Comment (Lau-Odoo) [inline events/overview/track_talks.rst:14]:
```suggestion
| Once *Website menu* is enabled, the options *Tracks on Website* and
  *Track Proposals on Website* become available. Enabling them adds the menus *Talks*,
  *Talk Proposals*, and *Agenda* to your website.
```

File: events/overview/track_talks.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+.. image:: media/track_proposals_website.png
+   :align: center
+   :alt: View of an event form and the options to track and propose speaker in Odoo Events
+
+From there, users see a list of the tasks and of the whole agenda, besides having the ability to
+propose their own talks.
```
Comment (Lau-Odoo) [inline events/overview/track_talks.rst:21]:
```suggestion
Besides having the ability to suggest their own talks, the website now allows users to see a list of the whole agenda and scheduled talks.
```

File: events/overview/track_talks.rst [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+.. image:: media/proposals_list.png
+   :align: center
+   :alt: View of the talks’ proposals page emphasizing the column proposal in Odoo Events
+
+If the proposal is accepted and can go online, simply open its form and click on *Go to Website*.
+From the website page, turn *Published* on.
+
+.. image:: media/publish_speakers.png
+   :align: center
+   :alt: View of the website page to publish a proposed talk for Odoo Events
```
Comment (Lau-Odoo) [inline events/overview/track_talks.rst:46]: I think you should zoom on the "unpublished".

File: events/overview/track_talks.rst [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+.. tip::
+   To handle tracks internally and not have the menus *Talks*, *Talk Proposals*, and *Agenda*
+   online, use the menu *Tracks* individually.
+
+Publishing speaker proposals
+----------------------------
+
+Once partners have filled in the appropriate form and submitted their proposals, a new *Proposal* is
+instantly created under the menu *Track* in your event form.
```
Comment (Lau-Odoo) [inline events/overview/track_talks.rst:35]:
```suggestion
instantly created under the menu *Track*.
```

File: events/overview/track_attendees.rst [hunk 8]
```diff
@@ -0,0 +1,47 @@
+====================
+Track your Attendees
+====================
+
+Attendees are at the heart of your event. Keeping track of its number and being able to manage their
+registration and attendance is necessary for planning and analyses reasons.
```
Comment (Lau-Odoo) [inline events/overview/track_attendees.rst:6]:
```suggestion
Attendees are the heart of your event. Keeping track of their number and being able to manage their
registration and attendance is necessary for planning and analyzes reasons.
```

File: events/overview/track_attendees.rst [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+Attendees list and attendance
+=============================
+
+| Once attendees have registered or bought their tickets, they are soon added to the *Attendees*
+  list.
```
Comment (Lau-Odoo) [inline events/overview/track_attendees.rst:12]:
```suggestion
| Once attendees have registered or bought their tickets, they are added to the *Attendees*
  list.
```

File: events/overview/tickets.rst [hunk 10]
```diff
@@ -0,0 +1,53 @@
+===============
+Selling Tickets
+===============
+
+Save time, automate processes, and give attendees options to choose from in order to have the best
+purchase and payment methods available.
```
Comment (Lau-Odoo) [inline events/overview/tickets.rst:6]:
```suggestion
Save time, automate processes, and give attendees options to choose from to have the best
purchase and payment methods available.
```

File: events/overview/tickets.rst [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+Configuration
+=============
+
+| Go to :menuselection:`Configuration --> Settings` and activate *Tickets* and *Online Ticketing*.
+| *Tickets* allow tickets to be sold :ref:`events/create_event/through_sales_orders`. *Online
+  Ticketing* allows for the sale of tickets to happen :ref:`events/create_event/through_the_website`.
```
Comment (Lau-Odoo) [inline events/overview/tickets.rst:13]:
```suggestion
  Ticketing* allows the sale of tickets to happen :ref:`events/create_event/through_the_website`.
```

File: events/overview/tickets.rst [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+   :align: center
+   :alt: View of the settings page for Odoo Events
+
+.. _events/create_event/through_sales_orders:
+
+Through Sales Orders
+--------------------
+
+On the *Sales* application, choose the event product you created and add it as a product line. A
+window pops-up allowing you to choose the event for which you want to create the sales order.
```
Comment (Lau-Odoo) [inline events/overview/tickets.rst:25]:
```suggestion
window pops-up, allowing you to choose the event for which you want to create the sales order.
```

File: events/overview/tickets.rst [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+      :alt: View of an event form highlighting the column product under the tab tickets in Odoo
+            Events
+
+.. _events/create_event/through_the_website:
+
+Through the Website
+-------------------
+
+On the website, once tickets are added to the cart, the user can continued the transaction choosing
+among the payment methods you chose to have available.
```
Comment (Lau-Odoo) [inline events/overview/tickets.rst:46]:
```suggestion
On the website, once tickets are added to the cart, the user can continue the transaction, choosing
among the payment methods you chose to have available.
```

File: events/overview/create_event.rst [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+*Unconfirmed* stage from which it can be edited and used again.
+
+.. image:: media/set_to_draft.png
+   :align: center
+   :alt: View of an event’s form emphasizing the option set to draft in Odoo Events
+
+Publish your event
+------------------
+
+*Go to Website*, and turn *Published* on.
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:90]:
```suggestion
*Go to Website* and turn *Published* on.
```

File: events/overview/create_event.rst [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+
+.. image:: media/publish_event.png
+   :align: center
+   :alt: View of a website page and the option to publish the event in Odoo Events
+
+.. seealso::
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:96]: "Track talks" is also linked to the creation of an event. So, why not put it here too?

File: events/overview/create_event.rst [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+content by dragging and dropping blocks.
+
+.. image:: media/website_edit.png
+   :align: center
+   :alt: View of an event's website page emphasizing the edit button for Odoo Events
+
+Tab: Tickets
+------------
+
+| Go to :menuselection:`Events --> Configuration --> Settings`, and enable
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:40]:
```suggestion
| Go to :menuselection:`Events --> Configuration --> Settings` and enable
```

File: events/overview/create_event.rst [hunk 17]
```diff
@@ -0,0 +1,98 @@
...
+  :doc:`Tickets and Online Ticketing <tickets>`.
+| Now, under the *Tickets* Tab, add lines to include the different ticket types you offer. Include
+  the price, the start and end date for registrations, and even the maximum number of tickets that
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:43]:
```suggestion
  the price, the start and end dates for registrations, and even the maximum number of tickets that
```

File: events/overview/create_event.rst [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+  can be sold.
+| The *Reserved Seats* and *Unconfirmed Seat Reservations* are counted as attendees are
+  :doc:`confirmed <track_attendees>` or not.
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:46]:
```suggestion
| The *Reserved Seats* and *Unconfirmed Seat Reservations* are counted as attendees are
  :doc:`confirmed <track_attendees>` or not.
```

I don't understand this sentence. You probably forgot a word or something :)

File: events/overview/create_event.rst [hunk 19]
```diff
@@ -0,0 +1,98 @@
...
+   :align: center
+   :alt: View of an event form emphasizing the tab tickets in Odoo Events
+
+Tab: Communication
+------------------
+
+| Choose the email and/or SMS message template, as well as the frequency with which you would like
+  to communicate with your attendees under the tab *Communication*.
+| Create a template on the fly or choose an existing one. Define the *Interval* (2, 7, 15..) for the
+  *Unit* (days, hours, weeks, etc.), and the *Trigger* action (after registration, after the event,
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:58]:
```suggestion
  *Unit* (days, hours, weeks, etc.) and the *Trigger* action (after registration, after the event,
```

File: events/overview/create_event.rst [hunk 20]
```diff
@@ -0,0 +1,98 @@
...
+
+.. image:: media/communication_tab.png
+   :align: center
+   :alt: View of an event form and the tab communication in Odoo Events
+
+Tab: Questions
+--------------
+
+| Go to :menuselection:`Configuration --> Settings` and enable *Questions*.
+| Now, under the tab *Questions*, edit your questions and answers choosing if you would like your
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:69]:
```suggestion
| Now, under the tab *Questions*, edit your questions and answers, choosing if you would like your
```

File: events/overview/create_event.rst [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+your audience, generate word of mouth about your company, allow company employees and attendees to
+network, and provide an opportunity to sales representatives walk their prospects through their
+pipelines. And, because your event means your rules, you need a solution that allows you to not just
+create and manage, but also customize and align your event's planning to your way of execution.
+
+Get started
+===========
+
+Under the menu *Events*, have a view of all existing events and some key information such as the
+number of expected and confirmed attendees.
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:15]:
```suggestion
Under the menu *Events*, have a view of all existing events and some key information, such as the
number of expected and confirmed attendees.
```

File: events/overview/create_event.rst [hunk 22]
```diff
@@ -0,0 +1,98 @@
+=======================
+Create your First Event
+=======================
+
+Some of the reasons why hosting events is important are: to create engaging experiences with
+your audience, generate word of mouth about your company, allow company employees and attendees to
+network, and provide an opportunity to sales representatives walk their prospects through their
+pipelines. And, because your event means your rules, you need a solution that allows you to not just
+create and manage, but also customize and align your event's planning to your way of execution.
```
Comment (Lau-Odoo) [inline events/overview/create_event.rst:9]: Pretty long but I like it! :+1:

File: events/overview/tickets.rst (hunk 10 above)
Comment (jubodoo) [inline events/overview/tickets.rst:6]: Sentence is hard to read again. 

If you automate process you save time, and the second part is weird. Rewrite it please. :)

File: events/overview/track_attendees.rst [hunk 23]
```diff
@@ -0,0 +1,46 @@
+====================
+Track your Attendees
+====================
+
+Attendees are at the heart of your event. Keeping track of their number and being able to manage
+their registration and attendance is necessary for planning and analyzes reasons.
```
Comment (jubodoo) [inline events/overview/track_attendees.rst:6]: Attendees are at the heart of your event. Keeping track of their number and managing their registration and attendance is necessary for planning and analyzing reasons.

File: events/overview/track_talks.rst [hunk 24]
```diff
@@ -0,0 +1,49 @@
+======================
+Track and Manage Talks
+======================
+
+Allow partners to submit talk proposals so they can share their knowledge and expertise. Good
+speakers and talks also add extra value to your event.
```
Comment (jubodoo) [inline events/overview/track_talks.rst:6]: Good speakers and talks are what makes your event great.

**Pull request merged**